src/plugins/event_annotation/public/get_table_list.tsx

```diff
@@ -34,28 +33,26 @@ export const getTableList = (
   services: EventAnnotationListingPageServices
 ) => {
   return (
-    <RootDragDropProvider>
```

Comment:
As discussed offline, this removal breaks the gated draggable dimension button interface (sorry—easy to miss 😅 ). We should fix this before merging!